_posts/2019-08-06-serverless-static-sites.md

```diff
@@ -0,0 +1,232 @@
+---
+title: "Bringing Serverless to a webpage near you with Hugo & Kubernetes"
+description: With the simplicity of OpenFaaS's templating system we can easily create new templates in no time. Today we will be showing how to continuosly deploy static sites using a new template for hugo and the FunctionIngress CRD for custom domain names.
+date: 2019-08-06
+image: /images/2019-serverless-static-sites/background.jpg
+categories:
+  - community
+  - end-user
+  - ingress
+  - kubernetes
+  - hugo
+author_staff_member: matias
+dark_background: true
+---
+
+Learn how you can continuosly deploy static sites with Hugo, OpenFaaS Cloud (or GitLab) and the FunctionIngress CRD.
+
+## Pre-req
+* Kubernetes
+
+You need to have a Kubernetes cluster up & running. I recommend [DigitalOcean](https://www.digitalocean.com/), it's cost-efficient and very easy to set-up.
+
+* [Install Helm](https://github.com/openfaas/faas-netes/blob/master/HELM.md)
+
+You'll need helm to install the components in the cluster. If you can't install `Tiller`, helm's server-side component, in your cluster then you can use `helm template`.
+
+* [Install OpenFaaS using Helm](https://github.com/openfaas/faas-netes/blob/master/chart/openfaas/README.md#install)
+
+For this tutorial you will need to install OpenFaaS using an ingress controller instead of exposing the gateway with a LoadBalancer. For this you'll need to set `ingress.enabled=true` and `exposeServices=false` when installing it.
+
+* Install an Ingress Controller
+
+Currently we support [Nginx](https://github.com/kubernetes/ingress-nginx), [Traefik](https://docs.traefik.io/user-guide/kubernetes/) and Zalando's [Skipper](https://opensource.zalando.com/skipper/kubernetes/ingress-controller/). We will be using Nginx for this tutorial:
```

Comment:
"Currently we support" - not sure about this claim.

How about: 

"We will be using the new IngressOperator component which has support for the following IngressControllers:"